Tasks/SqlAzureDacpacDeployment/Utility.ps1

```diff
@@ -19,81 +19,40 @@ function Get-AgentIPRange
 
     [hashtable] $IPRange = @{}
 
-    $sqlCmd = Get-Command -Name "SqlCmd.exe" -ErrorAction SilentlyContinue
-    if ($sqlCmd)
-    {
-        $sqlCmdArgs = "-S `"$serverName`" -U `"$sqlUsername`" -P `"$sqlPassword`" -Q `"select getdate()`""
+    $sqlCmd = Split-Path -Parent $MyInvocation.MyCommand.Path | Join-Path -ChildPath "sqlcmdfolder\SQLCMD.exe"
+    $env:SQLCMDPASSWORD = $sqlPassword
+    $sqlCmdArgs = "-S `"$serverName`" -U `"$sqlUsername`" -Q `"select getdate()`""
     
-        Write-Verbose "Reching SqlServer to check connection by running sqlcmd.exe $sqlCmdArgs"
+    Write-Verbose "Reching SqlServer to check connection by running sqlcmd.exe $sqlCmdArgs"
 
-        $ErrorActionPreference = 'Continue'
+    $ErrorActionPreference = 'Continue'
 
-        $sqlCmdPath = $sqlCmd.Path
-        ( Invoke-Expression "& '$sqlCmdPath' --% $sqlCmdArgs" -ErrorVariable errors -OutVariable output 2>&1 ) | Out-Null   
+    $sqlCmdPath = $sqlCmd.Path
+    ( Invoke-Expression "& '$sqlCmdPath' --% $sqlCmdArgs" -ErrorVariable errors -OutVariable output 2>&1 ) | Out-Null   
```

Comment:
In invoke-expression simply use $PSScriptRoot\SqlCmd\SqlCmd.exe instead of having extra variables etc